client/store/modules/course.js

```diff
@@ -67,6 +69,32 @@ getter(function revisions() {
     .sort((rev1, rev2) => rev2.createdAt - rev1.createdAt);
 });
 
+getter(function getConfig() {
+  return element => {
+    if (!element.type) return {};
+    // NOTE: teaching elements always have activityId foreign key and that is
+    // how we can tell if an element is tes or activity
+    const isTes = !!element.activityId;
+    if (isTes) {
+      const course = this.rootGetters['course/course'];
+      return getTesMeta(course.schema, element.type) || {};
+    }
+    return getLevel(element.type) || {};
```

Comment:
It would be nice if possible to add optional last param for default value :bulb:

Reply:
Hmm, idea was to do following:
```js
function getTesMeta(schemaId, type, defaultValue) {
  return find(getSchema(schemaId).tesMeta, { type }) || defaultValue;
}

getTesMeta(course.schema, element.type, {});
// and modify getLevel in same fashion:
getLevel(element.type, {});
```